build.gradle

```diff
@@ -204,117 +233,90 @@ repositories {
   }
 }
 
-def versions = [
-  junit           : '5.7.2',
-  junitPlatform	  : '1.7.2',
-  logging         : '5.1.7',
-  pitest          : '1.5.2',
-  restAssured     : '4.3.3',
-  serenity        : '3.1.20',
-  springDoc       : '1.6.6',
-  springCloud     : '3.1.0',
+def versions =[
+  springDoc: '1.6.6',
   springFoxSwagger: '3.0.0',
-  springHystrix   : '2.2.10.RELEASE',
-  tomcat          : '9.0.58'
-]
-
-ext {
-  log4JVersion = "2.17.1"
-}
-
-
-ext.libraries = [
-  junit5: [
-    "org.junit.jupiter:junit-jupiter-api:${versions.junit}",
-    "org.junit.jupiter:junit-jupiter-engine:${versions.junit}",
-    "org.junit.jupiter:junit-jupiter-params:${versions.junit}",
-    "org.junit.platform:junit-platform-commons:${versions.junitPlatform}",
-    "org.junit.platform:junit-platform-engine:${versions.junitPlatform}"
-  ]
+  springCloud: '2021.0.1',
```

Comment:
don't think this is right?

Reply:
The spring cloud version? They have a weird naming convention but this is the BOM version: https://spring.io/blog/2022/02/18/spring-cloud-2021-0-1-has-been-released